### DISCUSSION TITLE
[Run issue]: How to implement pH scaling?

### DISCUSSION THREAD

**Discussion by jonahpeter** [topas-nbio/TOPAS-nBio#11] [How to?]
Is pH scaling officially supported with TOPAS-nBio 3.0? Running the example `FrickeIRT.txt` yields a Gvalue of 0 at all time points, and the reactions appear to be defined based on an outdated version of nBio (for instance, there are still Type V reactions which I think have been deprecated).

Is there a working example of pH scaling (such as the one used to produce the regression testing results) that can be distributed? I could open a ticket for each issue I encounter but that seems very inefficient!

Many thanks!
Jonah

**Comment by JoseRamosMendez** [comment]
Hi there! Yes, the regression tests for TOPAS-nBio are public. [Here](https://github.com/topas-nbio/TOPAS-nBio-regression) you can find updated parameter files including pH scaling (Fricke)

**Reply by jonahpeter** [reply]
Wonderful, thanks Jose!

**Reply by jonahpeter** [reply]
Running the FrickeIRT example included with the regression tests seems to work, but I did notice that there are some reactions still defined in `depFile2.txt` as Type 5. I also noticed that the e_aq + e_aq reaction in the same file is defined as k = 0.55e10. However, I thought that for Type 1 and 3 reactions, there should be an extra factor of 2 when the reactants are identical, so that the constant given to TOPAS is 2k = 1.10e10.

Was the behavior for Type 5 reactions and/or reactions with identical reactants changed for TOPAS-nBio 3.0?

Thank you!!

**Reply by JoseRamosMendez** [reply]
Hi there,
You're absolutely right. We don't use type V as the spin considerations are embedded in the observed reaction rate. Thus, for e + e, k = 1.1e10, similar fo H+H. I remember @tmasilela did that change and uploaded the regression test (e.g., `GvalueIRT/ParameterFiles/depFile1.txt` has it), but it seems that this old file skipped the commit somehow.

**Reply by jonahpeter** [reply]
Excellent, thank you. And just to clarify, for Type 2 and 4 (e.g., OH + OH) there is no factor of 2 because kobs is calculated internally from the molar reaction rate, correct?

**Reply by JoseRamosMendez** [reply]
Correct!
